mono/mini/debugger-agent.c

```diff
@@ -99,7 +98,14 @@
 
 #include <mono/utils/mono-os-mutex.h>
 
+#ifdef IL2CPP_MONO_DEBUGGER
+#define THREAD_TO_INTERNAL(thread) thread
```

Comment:
I believe il2cpp has internal_thread too, but when running in .NET 4.6 profile mode.

Reply:
Yes, `internal_thread` does exist in `Il2CppThread` as well. We decided to use the point to the higher level `Il2CppThread` object though, as the `vm::Thread` APIs in libil2cpp use that pointer. So we don't need to keep a map from the `internal_thread` pointer back to the `Il2CppThread` pointer.